token/core/common/validator.go

```diff
@@ -93,23 +101,29 @@ func (v *Validator[P, T, TA, IA]) VerifyTokenRequestFromRaw(getState driver.GetS
 		signatures = tr.Signatures
 	}
 
+	attributes := make(map[string][]byte)
+	attributes[TokenRequestToSign], err = v.Serializer.MarshalTokenRequestToSign(req, nil)
+	if err != nil {
+		return nil, nil, errors.Wrap(err, "failed to marshal signed token request")
+	}
+
 	backend := NewBackend(getState, signed, signatures)
-	return v.VerifyTokenRequest(backend, backend, anchor, tr)
+	return v.VerifyTokenRequest(backend, backend, anchor, tr, attributes)
```

Comment:
Do we expect a variable number/type of attributes to be passed? otherwise we could pass a typed struct and if we see that it becomes too big, then we can switch to the map.